admin/views/pupil-status/view-pupil-status.ejs

```diff
@@ -314,24 +313,22 @@
                         </div>
                         <div class="app-step-nav__panel js-panel" id="step-panel-1">
                             <p class="app-step-nav__paragraph">
-                                <span class="govuk-!-font-weight-bold">Pupil check not received:</span> it may still be
-                                held on the device that the pupil sat the check on. If this device is known please go to
-                                the pupil check home page and it will try to recover the data if possible.
+                                <span class="govuk-!-font-weight-bold">Pupil check not received:</span> you may find the check is still be held on the device the pupil used to take the check . If this device is known, you should go to the  check home page and the device will attempt to recover the data if possible.
```

Comment:
This line has grown far too long.  Lines should be wrapped at something sensible.  There is a space before the full stop which needs removing.